lib/cli/commands-schema/no-service.js

```diff
@@ -30,6 +30,16 @@ commands.set('', {
       usage: 'Enable Serverless Console integration. See: http://slss.io/console',
       type: 'boolean',
     },
+    'dev': {
+      usage: 'Launch dev mode activity feed. See: http://slss.io/console',
+      type: 'boolean',
+    },
+    'function': {
+      usage:
+        'Launch dev mode activity feed focused on a specific function inf your serverless.yml file.',
```

Comment:
I think just this argument is not responsible for launching dev mode activity.

More appropriate description will probably be something as: _Name of a function to be observed in dev mode feed_